.github/workflows/test.yml

```diff
@@ -8,15 +8,11 @@ jobs:
     runs-on: ubuntu-latest
     steps:
     - uses: actions/checkout@v1
-    - run: |
-      swift build
-      swift test --enable-test-discovery --sanitize=thread
+    - run: swift test --enable-test-discovery --sanitize=thread
```

Comment:
@tanner0101 Why did this change? Now the example console app is not being built on CI anymore.

Reply:
`swift test` also builds the project.